src/main.ts

```diff
@@ -223,6 +225,13 @@ app.on("ready", async () => {
     debug("ready");
     initApp();
 
+    const store = container.get("store") as Store<RootState>;
+    store.dispatch(setLocale(
+        Object.keys(AvailableLanguages).reduce(
+            (pv, cv) => cv === app.getLocale().split("-")[0] ? cv : pv,
+            "en")
```

Comment:
Would it be possible to centralize the `en` default value? (with `src/common/redux/reducers/i18n.ts` `initialState:I18NState`.`locale`)

Reply:
Nit-picking: instead of a convoluted `reduce` call, could this simply be a `find()` array query?
```javascript
const loc = app.getLocale().split("-")[0];
const lang = Object.keys(AvailableLanguages).find((l) => { l === loc; }) | | "en"
```
?

Reply:
Or simpler even:
```
const loc = app.getLocale().split("-")[0];
const lang = Object.keys(AvailableLanguages).contains(loc) ? loc : "en";
```
:)